### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by mdzidic** [flapjack/flapjack#955]
Hello,
Do You have any plans to continue developing Flapjack or it's dead?
There is no updates for last two year and 125 issues at moment :(

**Comment by macropin** [comment]
Looks like we need an official fork and maintenance crew.

**Comment by ali-graham** [comment]
I've had plans, but little time to do much about them. Given that some of them involve changing the datastore used, those plans will probably be best carried out in a new project, so yeah, open to discussions on what Flapjack needs in maintenance terms to get both v1 and v2 behaving well.

**Comment by jessereynolds** [comment]
I'm in the same boat as @ali-graham ... plans but little time.

**Comment by mdzidic** [comment]
Any news guys?

**Comment by KonstantinShvaykovskiy** [comment]
We are interested too! Any news?

**Comment by jessereynolds** [comment]
Yes pretty much dead. I (for one) would love to work on it again but without any funding it’s not currently possible.

**Comment by jessereynolds** [comment]
We should leave this open until the project is revived or pronounced dead. ... Get in touch via jesse at va com au if you have ideas

**Comment by psyhomb** [comment]
This have been a great project really and I'm sad because it's been discontinued but for you guys who would like to use similar kind of tool with existing monitoring systems, please check it out [Alerta](https://alerta.io/) project.
